src/main/java/appeng/core/sync/packets/PacketValueConfig.java

```diff
@@ -59,8 +59,8 @@
 public class PacketValueConfig extends AppEngPacket
 {
 
-	public final String Name;
-	public final String Value;
+	private final String Name;
+	private final String Value;
```

Comment:
I know.. There are a bunch of them. Probably something I will tackle should the PR for private fields be merged. Also the whole `PacketValueConfig` is a bit cheesy. Something I want to address with the network refactoring.